src/cow-react/modules/limitOrders/pure/ReceiptModal/index.tsx

```diff
@@ -91,8 +92,16 @@ export function ReceiptModal({
             </styledEl.Field>
 
             <styledEl.Field>
-              <FieldLabel label="Execution price" tooltip={tooltips.EXECUTION_PRICE} />
-              <PriceField order={order} price={executionPrice} />
+              {order.fullyFilled ? (
+                <>
+                  <FieldLabel label="Execution price" tooltip={tooltips.EXECUTION_PRICE} />{' '}
+                  <PriceField order={order} price={executionPrice} />
+                </>
+              ) : (
+                <>
+                  <FieldLabel label="Executes at" tooltip={tooltips.EXECUTES_AT} />
+                </>
```

Comment:
do you need this extra Fragment brackets?

Reply:
Yes, this is a placeholder. We need to add the EXECUTES AT pricefield here.